docs/api-reference/core.md

```diff
@@ -2702,6 +2702,18 @@ int32
 <p>MaxGracefulTerminationSeconds is the number of seconds CA waits for pod termination when trying to scale down a node (default: 600).</p>
 </td>
 </tr>
+<tr>
+<td>
+<code>ignoreTaints</code></br>
+<em>
+[]string
+</em>
+</td>
+<td>
+<em>(Optional)</em>
+<p>IgnoreTaints specifies list of taint keys to ignore in node templates when considering to scale a node group</p>
```

Comment:
Nit - Same change at other places also.
```suggestion
<p>IgnoreTaints specifies a list of taint keys to ignore in node templates when considering to scale a node group</p>
```

Reply:
Thanks @acumino. I will make the changes.